docs/resources/ocean_aks_np.md

```diff
@@ -306,4 +306,49 @@ update_policy {
     node_names = ["aks-omnp123456-7890-vmss000001"]
   }
 }
+```
+<a id="scheduled-task"></a>
+## Scheduled Task
```

Comment:
scheduling is already mentioned in the document, remove whichever not required

Reply:
Removed scheduling description and example that was mentioned earlier and added complete scheduling block next to place of new field descriptions.